py/picca/io.py

```diff
@@ -184,91 +146,83 @@ def read_drq(drq_filename, z_min, z_max, keep_bal, bi_max=None):
             mjd: the Modified Julian Date of the observation
             fiberid: the fiberid of the observations
     """
-    hdul = fitsio.FITS(drq_filename)
+    userprint('Reading catalog from ', drq_filename)
+    catalog = Table(fitsio.read(drq_filename, ext=1))
+
+    keep_columns = ['RA', 'DEC', 'Z']
+    if 'desi' in mode:
+        obj_id_name='TARGETID'
+        catalog.rename_column('TARGET_RA', 'RA')
+        catalog.rename_column('TARGET_DEC', 'DEC')
+        keep_columns += ['TARGETID', 'TILEID', 'PETAL_LOC', 'NIGHT', 'FIBER']
```

Comment:
Yes. Could you send me a script where this is run on desi mocks? Or point me to someone that does that. I agree this is not covered by Travis and has to be added. 
I added 
`if 'desi' in mode and 'TARGETID' in catalog.colnames:`
which should cover the cases you mention.

Reply:
about the script, please remind me when NERSC is back,

about the change to `if 'desi' in mode and 'TARGETID' in catalog.colnames` this would keep things in the cat on eBOSS names, which I guess would be removed in `read_from_desi`, that's why I suggested the mapping

Reply:
ok, the current approach seems to work on with a drq-style desi mock catalog and the london mocks, so I'd consider this solved.